src/sdam/topology.ts

```diff
@@ -911,7 +917,7 @@ function drainWaitQueue(queue: List<ServerSelectionRequest>, err?: MongoDriverEr
           );
         }
       }
-      waitQueueMember.callback(err);
+      waitQueueMember.reject(err);
```

Comment:
maybe `drainError`, this is actually always going to be `MongoTopologyClosedError` at least currently, maybe something alluding to that `shutdownError`